StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/ScriptMessageHandler.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ScriptMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 5/1/24.
+//
+
+import WebKit
+
+/// Convenience class that conforms to WKScriptMessageHandler and can be instantiated with a closure
+class ScriptMessageHandler<T: Decodable>: NSObject, WKScriptMessageHandler {
```

Comment:
supernit: Could we use a more descriptive name than `T`? Perhaps `Payload`, `Args`, `PayloadType`, etc?